python/lvmdrp/core/sky.py

```diff
@@ -332,26 +346,39 @@ def skymodel_pars_from_header(header, telescope):
+        f"HIERARCH SKYMODEL SM_H": (sm_h.to(u.km).value, "observatory height in km"),
+        f"HIERARCH SKYMODEL SM_HMIN": ((2.0 * u.km).value, "lower height limit in km"),
+        f"HIERARCH SKYMODEL SCI ALT": ("{:.4f}".format(sci_alt.to(u.deg).value), "altitude of object above horizon (deg)"),
+        f"HIERARCH SKYMODEL SKYE ALT": ("{:.4f}".format(skye_alt.to(u.deg).value), "altitude of object above horizon (deg)"),
+        f"HIERARCH SKYMODEL SKYW ALT": ("{:.4f}".format(skyw_alt.to(u.deg).value), "altitude of object above horizon (deg)"),
+        f"HIERARCH SKYMODEL ALPHA": ("{:.4f}".format(alpha.to(u.deg).value), "separation of Sun and Moon from Earth (deg)"),
+        f"HIERARCH SKYMODEL SCI RHO": ("{:.4f}".format(sci_rho.to(u.deg).value), "separation of Moon and object (deg)"),
+        f"HIERARCH SKYMODEL SKYE RHO": ("{:.4f}".format(skye_rho.to(u.deg).value), "separation of Moon and object (deg)"),
+        f"HIERARCH SKYMODEL SKYW RHO": ("{:.4f}".format(skyw_rho.to(u.deg).value), "separation of Moon and object (deg)"),
+        f"HIERARCH SKYMODEL ALTMOON": ("{:.4f}".format(altmoon.to(u.deg).value), "altitude of Moon above horizon (deg)"),
+        f"HIERARCH SKYMODEL MOONDIST": ("{:.4f}".format(moondist.value), "distance to Moon (mean distance=1)"),
+        f"HIERARCH SKYMODEL PRES": ((744 * u.hPa).value, "pressure at observer altitude (hPa), set: 744"),
+        f"HIERARCH SKYMODEL SSA": (0.97, "aerosols' single scattering albedo, set: 0.97"),
+        f"HIERARCH SKYMODEL CALCDS": ( "N", "cal double scattering of moon (Y or N)"),
+        f"HIERARCH SKYMODEL O2COLUMN": (1.0, "relative ozone column density (1->258 DU)"),
+        f"HIERARCH SKYMODEL MOONSCAL": (1.0, "scaling factor for scattered moonlight"),
+        f"HIERARCH SKYMODEL SCI LON_ECL": ("{:.4f}".format(sci_lon_ecl.to(u.deg).value), "heliocen ecliptic longitude (deg)"),
+        f"HIERARCH SKYMODEL SCI LAT_ECL": ("{:.4f}".format(sci_lat_ecl.to(u.deg).value), "ecliptic latitude (deg)"),
+        f"HIERARCH SKYMODEL SKYE LON_ECL": ("{:.4f}".format(skye_lon_ecl.to(u.deg).value), "heliocen ecliptic longitude (deg)"),
+        f"HIERARCH SKYMODEL SKYE LAT_ECL": ("{:.4f}".format(skye_lat_ecl.to(u.deg).value), "ecliptic latitude (deg)"),
+        f"HIERARCH SKYMODEL SKYW LON_ECL": ("{:.4f}".format(skyw_lon_ecl.to(u.deg).value), "heliocen ecliptic longitude (deg)"),
+        f"HIERARCH SKYMODEL SKYW LAT_ECL": ("{:.4f}".format(skyw_lat_ecl.to(u.deg).value), "ecliptic latitude (deg)"),
+        f"HIERARCH SKYMODEL EMIS_STR": (",".join(map(str, [0.2])), "grey-body emissivity"),
+        f"HIERARCH SKYMODEL TEMP_STR": (",".join(map(str, [(290.0 * u.K).value])), "grey-body temperature (K)"),
+        f"HIERARCH SKYMODEL MSOLFLUX": (130.0, "monthly-averaged solar radio flux, set: 130"),
+        f"HIERARCH SKYMODEL SEASON": (season, "bimonthly period (1:Dec/Jan, 6:Oct/Nov.; 0:year)"),
+        f"HIERARCH SKYMODEL TIME": (time, "period of night (x/3 night, x=1,2,3; 0:night)"),
```

Comment:
Why are you formatting these to string here?  Do we need them as strings?  Why not keep the full precision and let users truncate them if needed?   They will be strings all the way to the drpall file, which is fine.  It may be confusing that all these numeric columns are strings, and not sure if the loss of precision matters.

Reply:
No longer strings (used np.round for formatting).  We discussed this as a group and agreed that we do not need full precision.